### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by nikhilgupta345**
If someone wants to help out with writing tests just lemme know which models you want to do (bids or transactions) and I can get the other one. Tests for Item isn't done yet but I'm just pushing this up now so you guys know the structure if you want to help.

### CODE REVIEW

**Comment by nikhilgupta345** [comment]
Also don't do the comments for each test case -- it's tedious and I'm removing mine lol

**Comment by srikanthchelluri** [comment]
sure i can do bids tomorrow

File: models/api/tests/tests_items.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
+from django.test import TestCase
+from django.urls import reverse
+
+from django.contrib.auth.models import User
+from api.models import Item, Bid, Transaction, UserData
+import json
+
+# Create your tests here.
+class ItemsTest(TestCase):
+	fixtures = ['test.json']
```
Comment (srikanthchelluri) [inline models/api/tests/tests_items.py]: why are you using test.json and not db.json? do we need a separate fixture

Comment (nikhilgupta345) [inline models/api/tests/tests_items.py]: Yeah basically to use fixtures in these tests you need a separate "fixtures" folder within each app. And then you create this Python list telling Django which files within that 'fixtures' folder you want to load before each test case.

File: models/api/views.py [hunk 2]
```diff
@@ -387,9 +381,9 @@ def bids_filter_by_query(request):
 	count = bids.count()
 	if count == 0:
```
Comment (nikhilgupta345) [inline models/api/views.py]: dont need this separate check for count, response will just be [] when looping through an empty queryset of bids

**Pull request merged**